packages/sync/src/modules/Terms.php

```diff
@@ -11,6 +11,14 @@ function name() {
 		return 'terms';
 	}
 
+	public function get_object_by_id( $object_type, $id ) {
```

Comment:
Nit: wanna add docs for this new method?

Reply:
Nice!